apps/Gruntfile.js

```diff
@@ -8,25 +8,32 @@ var logBuildTimes = require('./script/log-build-times');
 var webpackConfig = require('./webpack');
```

Comment:
Did you actually change Gruntfile.js or is this just a Prettier automated change?

Reply:
I didn't update the gruntfile at all. Should I commit again undoing the Prettier changes?